tests/test_on_data/test_gin_ecephys.py

```diff
@@ -113,17 +111,20 @@ class TestConverter(NWBConverter):
             data_interface=AxonaRecordingExtractorInterface,
             interface_kwargs=dict(file_path=str(DATA_PATH / "axona" / "axona_raw.bin")),
         ),
-        param(
-            data_interface=CEDRecordingInterface,
-            interface_kwargs=dict(file_path=str(DATA_PATH / "spike2" / "m365_1sec.smrx")),
-            case_name="smrx",
-        ),
         param(
             data_interface=EDFRecordingInterface,
             interface_kwargs=dict(file_path=str(DATA_PATH / "edf" / "edf+C.edf")),
             case_name="artificial_data",
         ),
     ]
+    if platform != "darwin" or version.parse(python_version()) >= version.parse("3.8"):
```

Comment:
Just FYI you could do all of this with the platform module:
```
if platform.system != "darwin" or  platform.python_version() > '3.8'
```
People consider `platform.system` more reliable than `sys.platform` but I don't think it matters in this case, so up to you. I think it is nice to use only library so it avoids some overloading of platform here.

Reply:
Going with https://stackoverflow.com/a/11674977 here for platform detection

Reply:
Is that thread and the linked comment that made me think that `platform.system` is more reliable? Did you get the opposite message from it?

Reply:
It is "the most reliable" part that I am not getting. My reading of the thread that you linked and the linked thread therein seems to suggest that `platform.system` is the most reliable:
https://stackoverflow.com/questions/1854/how-to-identify-on-which-os-python-is-running-on/58071295#58071295

I don't think it matters here though, and from reading above it does seem that `platform.system` does more than `sys.platform` in terms of computation. I just wanted to see if I was misreading something on the threads.

Reply:
Here is the source code for `platform.system`

```python
def uname():

    """ Fairly portable uname interface. Returns a tuple
        of strings (system, node, release, version, machine, processor)
        identifying the underlying platform.

        Note that unlike the os.uname function this also returns
        possible processor information as an additional tuple entry.

        Entries which cannot be determined are set to ''.

    """
    global _uname_cache

    if _uname_cache is not None:
        return _uname_cache

    # Get some infos from the builtin os.uname API...
    try:
        system, node, release, version, machine = infos = os.uname()
    except AttributeError:
        system = sys.platform
        node = _node()
        release = version = machine = ''
        infos = ()

    if not any(infos):
        # uname is not available

        # Try win32_ver() on win32 platforms
        if system == 'win32':
            release, version, csd, ptype = win32_ver()
            machine = machine or _get_machine_win32()

        # Try the 'ver' system command available on some
        # platforms
        if not (release and version):
            system, release, version = _syscmd_ver(system)
            # Normalize system to what win32_ver() normally returns
            # (_syscmd_ver() tends to return the vendor name as well)
            if system == 'Microsoft Windows':
                system = 'Windows'
            elif system == 'Microsoft' and release == 'Windows':
                # Under Windows Vista and Windows Server 2008,
                # Microsoft changed the output of the ver command. The
                # release is no longer printed.  This causes the
                # system and release to be misidentified.
                system = 'Windows'
                if '6.0' == version[:3]:
                    release = 'Vista'
                else:
                    release = ''

        # In case we still don't know anything useful, we'll try to
        # help ourselves
        if system in ('win32', 'win16'):
            if not version:
                if system == 'win32':
                    version = '32bit'
                else:
                    version = '16bit'
            system = 'Windows'

        elif system[:4] == 'java':
            release, vendor, vminfo, osinfo = java_ver()
            system = 'Java'
            version = ', '.join(vminfo)
            if not version:
                version = vendor

    # System specific extensions
    if system == 'OpenVMS':
        # OpenVMS seems to have release and version mixed up
        if not release or release == '0':
            release = version
            version = ''

    #  normalize name
    if system == 'Microsoft' and release == 'Windows':
        system = 'Windows'
        release = 'Vista'

    vals = system, node, release, version, machine
    # Replace 'unknown' values with the more portable ''
    _uname_cache = uname_result(*map(_unknown_as_blank, vals))
    return _uname_cache

### Direct interfaces to some of the uname() return values

def system():

    """ Returns the system/OS name, e.g. 'Linux', 'Windows' or 'Java'.

        An empty string is returned if the value cannot be determined.

    """
    return uname().system
```

As you can see, the initial value starts off as `sys.platform` and then gets mapped through a whole bunch of conditions, 99% of which (and this seems to hold for all secondary functions in the `platform` package) are for classifying configurations of Windows and Linux. The most they touch MacOS is to capitlize the `sys.platform` output of `"darwin"` to `"Darwin"`.

So again, for simply determining if I am specifically on (or not on, but don't care specifically what I am on) a Mac platform, all I need is `sys.platform`. If I wanted to know specifically the kind of device otherwise, yes I would use `platform.system`.

Reply:
Thank you for looking into this.
> As you can see, the initial value starts off as sys.platform

This is only true if ` os.uname()` does not work (which I read only works on Unix, so mostly read). 

It seems that the confusion from the linked thread is that they wrote this as python 3.7 but this function changed after, check it out:
https://github.com/python/cpython/blob/3.7/Lib/platform.py#L949